moveit_planners/pilz_industrial_motion_planner/src/trajectory_functions.cpp

```diff
@@ -571,8 +573,7 @@ bool pilz_industrial_motion_planner::isStateColliding(const bool test_for_self_c
   collision_detection::CollisionRequest collision_req;
   collision_req.group_name = group->getName();
   collision_detection::CollisionResult collision_res;
-  planning_scene::PlanningScene(robot_model).checkSelfCollision(collision_req, collision_res, *rstate);
-
```

Comment:
Ouch. Thanks for spotting this issue!

Reply:
Credit to @cambel for identifying the root cause :smiley: